### ISSUE TITLE
404 GET /metrics message

### ISSUE THREAD

**Issue by Liwellyen** [jupyterlab/jupyterlab#6884]
Since Jupyterlab version 1.0 I get messages like : 

[W 12:06:19.086 LabApp] 404 GET /metrics?1563962778910 

I working on a remote server. Only plotly extension and jupyerlab manager are installed. Before i instlaled jupyterlab 1.0 i uninstalled jupyterlab and jupyterlab-server and deleted all jupyterlab extensions in the share/jupyter/lab/extensions folder. Could it be that this was a unclean uninstallation? What could it else be?

**Comment by jtpio** [comment]
@Liwellyen does it also happen in a fresh conda environment?

This looks related the status bar memory indicator. But it shouldn't trigger the request if `nbresuse` is not installed.

**Comment by mayadebellis** [comment]
I am also getting this message. I created a clean conda environment and installed jupyterlab 1.0 etc. Afterwards I started trying to reinstall old extensions of mine and update them, so it is possible that I installed something that is corrupting it.

I have checked both pip and conda and `nbresuse` is not installed anywhere.

**Comment by Liwellyen** [comment]
@jtpio I don't use conda. I just installed jupyterlab Via pip3.

**Comment by jasongrout** [comment]
IIRC, this is from using an old notebook version. Newer notebook versions have that endpoint. It's a harmless error.

**Comment by jasongrout** [comment]
What notebook version do you have?

**Comment by mayadebellis** [comment]
I am on jupyter notebook version 6.0.0

**Comment by jasongrout** [comment]
@mayadebellis - what old extensions do you have installed? `jupyter labextension list`

**Comment by mayadebellis** [comment]
@jasongrout ahh it seems when I was seeing the error I didn't have my conda environment activated and it was using notebook version `5.5.0`. Thank you!

**Comment by wqh17101** [comment]
@jasongrout i got the same messages. I have installed the `nbresuse`. but it didnt work.
![image](https://user-images.githubusercontent.com/26429138/62446395-2e852a80-b795-11e9-8a6b-5f2ff3ccc94b.png)
Also i want to make `nbresuse` work

**Comment by wqh17101** [comment]
it seems that the `nbresuse` has conflict with `git` ,  if one of them 200, the other is always 404.
![image](https://user-images.githubusercontent.com/26429138/62447452-f7fcdf00-b797-11e9-9086-7424cec5f6da.png)
![image](https://user-images.githubusercontent.com/26429138/62447466-01864700-b798-11e9-8c42-83bbb1dff7a9.png)
Maybe both port of them is 80 cause this?
i dont know it is the confilct between these extensions or any two extensions.

**Comment by memory-yancy** [comment]
The same issue use `continuumio/anaconda3:5.0.0` docker image to test, and installed `nbresuse`  to test again, too

Steps:
```
[yancy@localhost UserManager]$ docker run -ti --rm --network host -v /home/yancy/.pip:/root/.pip continuumio/anaconda3:5.0.0 /bin/bash
root@localhost:/# pip install jupyterlab==1.1.2

.... ...
Successfully installed attrs-19.1.0 jinja2-2.10.1 json5-0.8.5 jsonschema-3.0.2 jupyterlab-1.1.2 jupyterlab-server-1.0.6 pyrsistent-0.15.4 six-1.12.0
root@localhost:/# jupyter-lab --ip 0.0.0.0 --allow-root --LabApp.token=''
... ....

[W 10:06:20.785 LabApp] 404 GET /metrics?1568109980781 (127.0.0.1) 1.51ms referer=http://127.0.0.1:8888/lab
[W 10:06:25.790 LabApp] 404 GET /metrics?1568109985786 (127.0.0.1) 1.33ms referer=http://127.0.0.1:8888/lab
```

**Comment by rsokl** [comment]
I hit this too from a clean conda environment on Windows 10:

```shell
> conda create -n jlab-test2 --override-channels --strict-channel-priority -c conda-forge -c anaconda jupyterlab
> conda activate jlab-test2
> jupyter lab
```

Upon creating a new notebook, the kernel never loads:

```shell
[I 12:14:29.551 LabApp] Kernel started: 844ecb27-bdd8-476a-b3ff-589427f971c9
[W 12:14:30.972 LabApp] 404 GET /metrics?1575738870969 (::1) 1.02ms referer=http://localhost:8890/lab
```

I tried this process both with and without admin privileges.

**Comment by unnir** [comment]
the same issue

**Comment by lambdamusic** [comment]
I'm not using Conda but still have this problem. 

I've also tried disabling all extensions but no luck. 

```
>jupyter labextension list
JupyterLab v1.2.6
Known labextensions:
   app dir: /usr/local/share/jupyter/lab
        @aquirdturtle/collapsible_headings v0.3.3 disabled OK
        @jupyterlab/git v0.7.0 disabled OK
        @jupyterlab/toc v1.0.1 disabled OK
        jupyterlab-recents v1.0.1 disabled OK
        jupyterlab_filetree v0.2.1 disabled OK
```

After running it, it all seems fine but then I quickly start getting 404s.. 

```
>jupyter lab
[I 12:25:14.210 LabApp] JupyterLab extension loaded from /usr/local/lib/python3.7/site-packages/jupyterlab
[I 12:25:14.211 LabApp] JupyterLab application directory is /usr/local/share/jupyter/lab
[I 12:25:14.443 LabApp] Serving notebooks from local directory: /Users/me/Notebooks
[I 12:25:14.443 LabApp] 0 active kernels
[I 12:25:14.443 LabApp] The Jupyter Notebook is running at:
[I 12:25:14.443 LabApp] http://localhost:8888/?token=xxx
[I 12:25:14.443 LabApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 12:25:14.446 LabApp]

    Copy/paste this URL into your browser when you connect for the first time,
    to login with a token:
        http://localhost:8888/?token=xxx
OneAuth Version 0.14.0 (com.microsoft.Edge)
[I 12:25:15.562 LabApp] Accepting one-time-token-authenticated connection from ::1
[W 12:25:16.346 LabApp] 404 GET /metrics?1581938716316 (::1) 29.01ms referer=http://localhost:8888/lab
[I 12:25:16.920 LabApp] Build is up to date
[W 12:25:21.360 LabApp] 404 GET /metrics?1581938721353 (::1) 3.14ms referer=http://localhost:8888/lab
[W 12:25:26.367 LabApp] 404 GET /metrics?1581938726361 (::1) 2.06ms referer=http://localhost:8888/lab
[W 12:25:31.379 LabApp] 404 GET /metrics?1581938731373 (::1) 2.11ms referer=http://localhost:8888/lab
[W 12:25:36.390 LabApp] 404 GET /metrics?1581938736384 (::1) 2.24ms referer=http://localhost:8888/lab
[W 12:25:41.401 LabApp] 404 GET /metrics?1581938741395 (::1) 1.86ms referer=http://localhost:8888/lab
[W 12:25:46.409 LabApp] 404 GET /metrics?1581938746403 (::1) 1.90ms referer=http://localhost:8888/lab
..etc...
```

Maybe this problem gets generated by the main jupyterlab process? I'm a bit stuck on how to debug it further...

**Comment by stefansimik** [comment]
The same problem here - using clean jlab-test environment on Win10 + Chrome latest.

**Comment by sanbeichahegongheguo** [comment]
The same problem here - using clean conda environment on Win10 + Chrome latest.

**Comment by hassanhub** [comment]
The same problem here - using Conda environment on Ubuntu 18.04 + Safari

**Comment by remborg** [comment]
One of my coworker had this issue when migrating to Jupyter Lab 2.1 (from 1.1.3).
It was an issue with Tornado and Notebook (used by pypi) that wouldn't talk to one another without being similar versions.
He fixed the issue by upgrading them.

**Comment by remborg** [comment]
One of my coworker had this issue when migrating to Jupyter Lab 2.1 (from 1.1.3).
In the logs we could see that there was no active kernels:
`[I 12:25:14.443 LabApp] 0 active kernels`

He found out that it was an issue with Tornado and Notebook (used by pypi) that wouldn't talk to one another without being similar versions and fixed the issue with and upgrade:
```
pip install notebook --upgrade
pip install tornado --upgrade
```

**Comment by zwelz3** [comment]
Thanks @remborg. Tornado was up-to-date for me, but `pip install notebook --upgrade` worked perfectly.

**Comment by jasongrout** [comment]
This is now fixed in JupyterLab 3.0, from https://github.com/jupyterlab/jupyterlab/pull/9386.
